### PR TITLE
Perceiver classification on OAI AKOA Image dataset s4580127

### PR DESCRIPTION
**Pull request by StopTheBoolean**
This PR contains files showcasing how to classify laterality the OAI AKOA Image dataset using the Perceiver transformer with a test accuracy of 98%.

Included are the following files:
- driver.py (driver script)
- Perceiver.py (tensorflow implementation of Perceiver)
- Example_Image.png (example image from OAI AKOA Dataset)
- Example_Image2.png (example image from OAI AKOA Dataset)
- training_plot.png (example plot of typical training cycle)
- perceiver_architecture.png (Image showcasing the model architecture)
- README.md

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
Difficulty Hard

Functionality: OK


Code
- Structure:  OK
- .py filenames should be lowercase

README 
- Content: missing references


PR Target OK

General Comments
- Commit messages: OK
- Please put pngs in a separate folder

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
No Header blocks -1
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage
Difficulty: Hard
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
No feedback required
Request Description OK
